build.yaml

```diff
@@ -3789,6 +3789,10 @@ steps:
     alwaysRun: true
     script: |
       set -e -o pipefail
+      curl -L https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-arm64 -o /usr/bin/jq
+      chmod +x /usr/bin/jq
+      curl -L "https://dl.k8s.io/release/v1.31.0/bin/linux/arm64/kubectl" -o /usr/bin/kubectl
+      chmod +x /usr/bin/kubectl
```

Comment:
😬  Is this really the official way to install `jq` and `kubectl`? Is there any way to apt-get install or something that would at least be using a signed download?

Reply:
... or could we pin to an azure-cli version which does have them installed?

Reply:
[As far as I can tell](https://github.com/microsoft/azurelinux), the only package manager that Azure Linux has is `rpm`, so I think we would have to download a `.rpm` file and install from that. That would be signed, I think? Very fair to want to avoid `curl` installing stuff

Reply:
I'm wary of pinning to an old `azure-cli` version though, I think we probably want updates when they're available and the old versions are no longer being supported, ostensibly.

Reply:
Wait apparently it has `yum`!